### PR TITLE
Add RLO CI tests

### PR DESCRIPTION
**Pull request by ryotatomioka**

### CODE REVIEW

File: test/builds/build_and_test.yml [hunk 1]
```diff
@@ -133,3 +134,209 @@ jobs:
 
   - script: rm -rf *
     displayName: 'Clean'
+
+
+- job: 'RLO_Ubuntu'
```
Comment (acl33) [inline test/builds/build_and_test.yml:139]: Is it possible to "#include" the corresponding file from rlo/test/builds ? It'll change with e.g. ksc Expr

File: rlo/scripts/azureml_util.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def combined_source_directory():
         )
         # Copy ksc in as a subdirectory. Note this does not include ts2ks.
         shutil.copytree(
-            os.path.join(root_dir, "knossos-ksc", "src", "python", "ksc"),
+            os.path.join(os.path.dirname(root_dir), "src", "python", "ksc"),
```
Comment (acl33) [inline rlo/scripts/azureml_util.py:32]: Perhaps rename "root_dir" to "rlo_dir"

**Review by acl33 — COMMENTED** [review]
To be honest I am thinking that I'm in favour of moving
rlo/src/rlo --> src/python/rlo
rlo/src/*.py --> src/python/*.py or src/python/rlo_scripts/*.py or something
rlo/test/rlo/ --> test/python/rlo/
etc.

but in the meantime, this look good to me!

**Comment by awf** [comment]
> To be honest I am thinking that I'm in favour of moving
> rlo/src/rlo --> src/python/rlo
> rlo/src/_.py --> src/python/_.py or src/python/rlo_scripts/*.py or something
> rlo/test/rlo/ --> test/python/rlo/
> etc.
> 
> but in the meantime, this look good to me!

Yes, and then even move `src\python\folder` to `src\folder` -- we don't have `src\haskell`..

File: rlo/scripts/azureml_util.py (hunk 2 above)
Comment (awf) [reply rlo/scripts/azureml_util.py:32]: Does `utils.py:get_ksc_dir` help here?

Comment (awf) [reply rlo/scripts/azureml_util.py:32]: Do we even need to copy ksc if rlo is in ksc?

**Review by awf — APPROVED** [review]
Great, thanks.

**Comment by ryotatomioka** [comment]
I will merge this first and then we can do another PR on moving files.

**Pull request merged**